src/tests/system/tests/test_ldap.py

```diff
@@ -241,89 +257,36 @@ def test_ldap__change_password_when_ldap_pwd_policy_is_set_shadow(client: Client
     client.sssd.domain["ldap_chpass_update_last_change"] = "True"
     client.sssd.start()
 
-    # Password is expired, change it
-    assert client.auth.parametrize(method).password_expired("tuser", "Secret123", "Redhat@321")
-
-    # Authenticate with new password
-    assert client.auth.parametrize(method).password("tuser", "Redhat@321")
-
-
-@pytest.mark.importance("medium")
-@pytest.mark.ticket(bz=1928648)
-@pytest.mark.topology(KnownTopology.LDAP)
-def test_ldap__network_timeout_parameters_shown_in_logs(client: Client, ldap: LDAP):
```

Comment:
Why removing this test? it seems to be a customer scenario.

Reply:
It is a customer scenario, I don't think it should've been automated in the first place. The system tests should be functional. If we do want to keep the test, test_logging.py was created just for log assertion tests and should be moved there.

Reply:
OK